integreat_cms/cms/templates/contacts/contact_list_archived_row.html

```diff
@@ -0,0 +1,45 @@
+{% load i18n %}
+{% load rules %}
+<tr class="border-t border-solid border-gray-200 hover:bg-gray-100">
+    <td class="pl-2">
+        <a href="" class="block py-3 px-2 text-gray-800">
+            {{ contact.title }}
+        </a>
+    </td>
+    <td>
+        <a href="" class="block py-3 px-2 text-gray-800">
+            {{ contact.name }}
+        </a>
+    </td>
+    <td>
+        <a href="" class="block py-3 px-2 text-gray-800">
+            {{ contact.poi }}
+        </a>
+    </td>
+    <td class="pr-2">
+        <a href="" class="block py-3 px-2 text-gray-800">
+            {{ contact.email }}
+        </a>
+    </td>
+    <td class="pr-2">
+        <a href="{{ contact.website }}"
+           target="_blank"
+           rel="noopener noreferrer"
+           class="text-blue-500 hover:underline"
+           title="{% translate "To website" %}">
+            {{ contact.website }}
+        </a>
+    </td>
+    <td class="pl-2 pr-4 text-right min">
+        {% has_perm 'cms.change_contact' request.user as can_edit_contact %}
+        {% if can_edit_contact %}
```

Comment:
```suggestion
        {% if perms.cms.change_contact %}
```
I think we can simply do like .

Maybe an delete option here too? We have it in the archived poi/event list.